asmcomp/power/emit.mlp

```diff
@@ -416,15 +422,26 @@ module BR = Branch_relaxation.Make (struct
         else 5
     | Lop(Istackoffset _) -> 1
     | Lop(Iload {memory_chunk; addressing_mode; _ }) ->
```

Comment:
`is_atomic` results in a fair amount of extra code being emitted, could you also take this into account? (I suppose that atomic loads are rare enough and that the specific bug you are hunting does not involve them, but the code is clearly incorrect in that case, right?)

Reply:
Oh, right, good catch, I had missed this.